desktop/extensions-sdk/extensions/publish.md

```diff
@@ -30,9 +35,48 @@ These guidelines don't replace Docker's terms of service or guarantee approval.
 - Test your extension on various platforms (Mac, Windows, Linux)
 - Read the [Terms of Service](https://www.docker.com/legal/extensions_marketplace_developer_agreement/){: target="_blank" rel="noopener" class="_" }
 
-### After you submit
+### Which publishing option to choose
+
+When submitting an extension from the extensions submissions [repository](https://github.com/docker/extensions-submissions/issues/new/choose){:target="_blank" rel="noopener" class="_" }, you have two options:
+- Self Published extensions
+- Docker Reviewed extensions
+
+Whether you choose one option or the other, the publishing process will be different.
```

Comment:
I might suggest "Depending on which option you select, the publishing process will differ."